client/me/concierge/book/confirmation-step.js

```diff
@@ -25,7 +25,7 @@ class ConfirmationStep extends Component {
 		return (
 			<Confirmation
 				description={ translate(
-					'We will send you an email with information on how to get prepared.'
+					'We will send you a calendar invitation and an email with information on how to prepare.'
```

Comment:
These two strings are identical. Is it possible to extract it into a reusable piece?

Reply:
I guess we could add it to the `Confirmation` component, but we had it that way before and needed to extract it out. For now I think it's fine maybe we will update the copy for reschedule.